libraries/botbuilder-azure-blobs/src/blobsStorage.ts

```diff
@@ -20,6 +26,10 @@ export interface BlobsStorageOptions {
     storagePipelineOptions?: StoragePipelineOptions;
 }
 
+function isCredentialType(value: any): value is TokenCredential {
```

Comment:
There's a method we can use in TokenCredential to check this.
![image](https://github.com/southworks/botbuilder-js/assets/44245136/d495dded-b71f-429e-8ea8-092efbc1044b)